toolbox/fdc3-for-web/demo/src/client/ui/drag.ts

```diff
@@ -25,7 +25,7 @@ export function dragElement(drag: HTMLElement, selector: HTMLElement, position:
   }
 
   function dragMouseDown(e: MouseEvent) {
-    console.log('astarting');
+    console.log('starting');
```

Comment:
Should we leave this `console.log` line? If so, can it be a `console.debug`, maybe with a more informative message? Like `console.debug('Channel Selector - DragMouseDown starting')`

Reply:
👍